### ISSUE TITLE
Function call adds extra newline

### ISSUE THREAD

**Issue by cblp** [chrisdone/hindent#243]
Snippet for TESTS.md:
```haskell
x =
  f (alpha, beta, gamma, delta, epsilon, zeta, eta, theta, iota, kappa, lambda, )
```

Test result:

       expected: 
       x =
         f (alpha, beta, gamma, delta, epsilon, zeta, eta, theta, iota, kappa, lambda, )
        but got: 
       x =
         f
           (alpha, beta, gamma, delta, epsilon, zeta, eta, theta, iota, kappa, lambda, )
       The diff:
       2c2,3
       <   f (alpha, beta, gamma, delta, epsilon, zeta, eta, theta, iota, kappa, lambda, )
       ---
       >   f
       >     (alpha, beta, gamma, delta, epsilon, zeta, eta, theta, iota, kappa, lambda, )

**Comment by chrisdone** [comment]
That's just line-breaking because the line is long. I think if there's an issue here, it's that `TupleSection` doesn't line-break: https://github.com/chrisdone/hindent/blob/master/src/HIndent/Pretty.hs#L462

**Comment by cblp** [comment]
1. It's unnecessary and wrong line break.
2. It is not connected with TupleSections.

        expected: 
        x =
          f (gamma, delta, epsilon, zeta, eta, theta, iota, kappa, lambda, mu, nu, xi, omicron)
         but got: 
        x =
          f
            (gamma, delta, epsilon, zeta, eta, theta, iota, kappa, lambda, mu, nu, xi, omicron)
        The diff:
        2c2,3
        <   f (gamma, delta, epsilon, zeta, eta, theta, iota, kappa, lambda, mu, nu, xi, omicron)
        ---
        >   f
        >     (gamma, delta, epsilon, zeta, eta, theta, iota, kappa, lambda, mu, nu, xi, omicron)

**Comment by chrisdone** [comment]
`f` line breaks because its argument doesn't fit on one line or within 80 columns. 

Do you want to add a special case that if `length f + 1 < indentSize` then don't line-break?

**Comment by cblp** [comment]
If it doesn't fit in line, items are aligned vertically:
```haskell
       x =
         f
           ( gamma
           , delta
           , epsilon
           , zeta
           , eta
           , theta
           , iota
           , kappa
           , lambda
           , mu
           , nu
           , xi
           , omicron
           , pi)
```

If I see horizontal tuple, obviously it fits into line. It look so.

It looks as it fits, but its length is actually 87. Maybe _this_ is the problem?

**Comment by chrisdone** [comment]
The problem is as I said above, the Tuple and TupleSections are not calculating their size properly and line-breaking early enough. That's why your 87 example doesn't line break the tuple, even though it should.

**Comment by cblp** [comment]
1. Can you elaborate? What is improper in tuple size calculation and what would be proper variant?
2. Does the ticket describing this exist?

**Comment by cblp** [comment]
Another example:

```haskell
x =
  f (alpha beta gamma delta epsilon zeta eta theta iota kappa lambda mu nu xi omicron)
```

Neither tuples not tuple sections involved.

**Comment by cblp** [comment]
I didn't understand why did you close this bug. But I fixed it, please see #254.
